defaults/main.yml

```diff
@@ -119,7 +119,7 @@ rhel_07_020100: true
 rhel_07_020101: true
 rhel_07_020110: true
 rhel_07_020240: true
-rhel_07_020260: true
+rhel_07_020260: false
```

Comment:
we just turned off the test - is there a way to fix it ?

Reply:
Go through the YUM source code, find the literal string and replace it with the the desired string, recompile it and use the new version.

Reply:
@aheifetz I'm assuming this is now not a blocker to approval & approving